app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/_module.less

```diff
@@ -397,7 +397,7 @@
 
     .box-tocart {
         &:extend(.abs-box-tocart all);
-        
+
```

Comment:
I don't thing this break line should be a part of this PR.

Reply:
Line 400 has new line in the original file. Not sure why it appears like this as diff.
![image](https://user-images.githubusercontent.com/34317830/68070576-96d1b100-fd70-11e9-8ad5-e3d0d8a4dce1.png)

Reply:
Alright.